File: packages/aws-cdk/lib/init-templates/sample-app/csharp/src/%name.PascalCased%/Program.template.cs
using Amazon.CDK;

namespace %name.PascalCased%
{
    sealed class Program
    {
        public static void Main(string[] args)
        {
            var app = new App();
            new %name.PascalCased%Stack(app, "%stackname%");

            app.Synth();
        }
    }
}

File: packages/aws-cdk/lib/init-templates/sample-app/fsharp/src/%name.PascalCased%/Program.template.fs
open Amazon.CDK
open %name.PascalCased%

[<EntryPoint>]
let main _ =
    let app = App(null)

    %name.PascalCased%Stack(app, "%stackname%", StackProps()) |> ignore

    app.Synth() |> ignore
    0
